pacu/settings.py

```diff
@@ -14,7 +14,9 @@
 
 ERROR_LOG_VERBOSITY = 'minimal'
 
-_home_dir = Path('~/.local/share/pacu')
+# TODO: Restore this to the original path 
+#_home_dir = Path('~/.local/share/pacu')
+_home_dir = Path('~/.local/share/pacu_vectra')
```

Comment:
This change should be removed.

Reply:
I have this here so people can use and/or test this version alongside stock Pacu. Without this line, anyone who has already run Pacu will encounter an error when running this version due to database schema mismatch. 

I was planning on removing it when we're ready to make it a PR against the main Pacu repo. Does that plan sound reasonable?

Reply:
Removed in 4b6a4ec